docs/en/operations/table_engines/mergetree.md

```diff
@@ -50,11 +51,13 @@ For a description of request parameters, see [request description](../../query_l
 
 - `ENGINE` - Name and parameters of the engine. `ENGINE = MergeTree()`. `MergeTree` engine does not have parameters.
 
-- `ORDER BY` — Primary key.
+- `ORDER BY` — Primary key (or sorting key if the separate `PRIMARY KEY` clause is present).
 
     A tuple of columns or arbitrary expressions. Example: `ORDER BY (CounterID, EventDate)`.
 If a sampling expression is used, the primary key must contain it. Example: `ORDER BY (CounterID, EventDate, intHash32(UserID))`.
 
+- `PRIMARY KEY` - Primary key if it differs from the [sorting key](mergetree.md#table_engines-mergetree-sorting_key) (the sorting key in this case is specified by the `ORDER BY` clause).
```

Comment:
I think it'd be better to break the cyclic dependency and just say that 1) by default primary key is equal to sorting key 2) few lines above ORDER KEY is just a sorting key without forward reference to PRIMARY KEY.